src/robot/variables/scopes.py

```diff
@@ -185,6 +185,8 @@ def _set_cli_variables(self, settings):
     def _set_built_in_variables(self, settings):
         for name, value in [('${TEMPDIR}', abspath(tempfile.gettempdir())),
                             ('${EXECDIR}', abspath('.')),
+                            ('&{SETTINGS}',    {"EXCLUDE_TAGS": settings.suite_config['exclude_tags'], 
+                                                "INCLUDE_TAGS": settings.suite_config['include_tags']}),
```

Comment:
Instead of exposing exclude and include as lists, it would probably be better to expose them as `robot.model.Tags` objects. `Tags` is a list-like object with these benefits in this context:
- Tags are matched in normalized manner. For example, `'e' in Tags(['E']) ` is true . This behavior would then also work in RF date when using e.g. `Should Contain` or `IF` expressions with include and exclude.
- Also patterns work in matching. For example, `'e*' in Tags(['example'])` and `'a AND b' in Tags(['a', 'b'])` are both true.
- Duplicate tags are automatically removed and tags are sorted.

I consider especially the first one pretty important, because otherwise users would need to implement normalization themselves to support tags similarly as Robot itself does.

The only downside of using `Tags` I can think of is that they are always unequal with lists. For example, both `Tags(['example']) ==['example']` and `Tags([]) == []` would both be false. It probably would be fine to change `Tags.__eq__` to handle this situation, though.

Reply:
Need to decide what format to use with key names. I think it would be best to use same names we use on command line i.e. `exclude` instead of `EXCLUDE_TAGS`. That makes the names easiest for users to remember.